src/mbgl/util/bounding_volumes.cpp

```diff
@@ -213,13 +213,15 @@ IntersectionResult Frustum::intersects(const AABB& aabb) const {
 
     bool fullyInside = true;
 
+    const auto epsilon = 1e-10;
```

Comment:
`std::numeric_limits<double>::epsilon()` was too small.

Maybe we should define this globally somewhere.

Reply:
I think it's fine for now, if we start having to do this in other places we should add some test constants though.